docs/troubleshoot/cli/troubleshoot-cli-credentials.md

```diff
@@ -11,3 +15,42 @@ Zowe CLI accepts various methods, or mechanisms, of authentication when communic
 However, some services can accept multiple methods of authentication. When multiple methods are provided (in a profile or command) for a service, the CLI follows an *order of precedence* to determine which method to apply.
 
 To find the authentication methods used for different services and their order of precedence, see the table in [Authentication mechanisms](../../extend/extend-cli/cli-devTutorials.md#authentication-mechanisms).
+
+### PEM certificate files
+
+PEM certificate files are one of the authentication mechanisms used by Zowe CLI.
```

Comment:
Should we state that a client certificate must be recorded in the mainframe service's trust store or keyring before the client certificate will be accepted by the mainframe service?

Should we state that the Zowe client certificate is a separate certificate from the certificate used by the APIML gateway to authenticate to the APIML service?

Reply:
updated the content accordingly -- thank you!